src/bin/parser_test.rs

```diff
@@ -31,34 +31,35 @@ fn main() -> Result<()> {
         tests_failed: Vec::new(),
     };
 
-    let filenames = Some(&["tests3.dat"][..]);
+    let filenames = Some(&["tests16.dat"][..]);
     let fixtures = testing::tree_construction::fixtures(filenames).expect("fixtures");
 
     for fixture_file in fixtures {
         println!(
-            "🏃‍♂️ Running {} tests from 🗄️ {:?}\n",
+            "\
+            🏃‍♂️ Running {} tests from 🗄️ {:?}",
             fixture_file.tests.len(),
             fixture_file.path
         );
 
         let mut test_idx = 1;
         for test in fixture_file.tests {
-            if test_idx == 46 {
-                run_tree_test(test_idx, &test, &mut results);
-            }
+            // if test_idx == 195 {
+            run_tree_test(test_idx, &test, &mut results);
+            // }
             test_idx += 1;
         }
-    }
 
-    println!(
-        "\
-🏁 Tests completed: Ran {} tests, {} assertions, {} succeeded, {} failed ({} position failures)",
-        results.tests,
-        results.assertions,
-        results.succeeded,
-        results.failed,
-        results.failed_position
-    );
+        println!(
+            "\
+    🏁 Tests completed: Ran {} tests, {} assertions, {} succeeded, {} failed ({} position failures)\n",
+            results.tests,
+            results.assertions,
+            results.succeeded,
+            results.failed,
+            results.failed_position
+        );
```

Comment:
Your statement is indented now, but I'm not seeing it indented.  Something I don't fully understand is going on (not that it's important).

![2023-10-25_17-22](https://github.com/gosub-browser/gosub-engine/assets/760949/9ee6f410-2bbe-45bc-acf4-f842d691fdf8)